### ISSUE TITLE
Server provides wrong answers

### ISSUE THREAD

**Issue by brnikita** [FosterFlow/fosterflow#202]

**Comment by brnikita** [comment]
![Image](https://github.com/FosterFlow/fosterflow/assets/761219/53c7fede-2273-44d0-b2fd-a7cf23ca6976)

**Comment by brnikita** [comment]
https://fosterflow.com/chats/208


![Image](https://github.com/FosterFlow/fosterflow/assets/761219/f1ebcae5-06ad-4d63-b6f1-0ca8e6ebf454)

**Comment by egortaran** [comment]
The context has been corrected in the commit 5e18661. 

In the official documentation, creating a response looks this:
```
response = openai.ChatCompletion.create(
    model="gpt-3.5-turbo",
    messages=[
        {"role": "system", "content": "You are a helpful assistant."},
        {"role": "user", "content": "Who won the world series in 2020?"},
        {"role": "assistant", "content": "The Los Angeles Dodgers won the World Series in 2020."},
        {"role": "user", "content": "Where was it played?"}
    ]
)
```

In our project we made:
```
response = openai.ChatCompletion.create(
    model="gpt-3.5-turbo",
    messages=messages,
    stream=True
)
```
were messages looks this:
```
[
    {'role': 'system', 'content': 'You are a helpful assistant.'}, 
    {'role': 'user', 'content': 'Hello'}, 
    {'role': 'assistant', 'content': 'Hello! How can I assist you today?'},
    {'role': 'user', 'content': '10+5'}, 
    {'role': 'assistant', 'content': '10 + 5 equals 15.'}, 
    {'role': 'user', 'content': '+8'},
]
```
And after we have this answer `{'role': 'assistant', 'content': 'To what number would you like to add 8? Could you please provide the number?'}`
